ui/src/jobs/components/__tests__/__snapshots__/ContactList.test.jsx.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ContactList renders with contacts with both email and name 1`] = `
+<div
+  className="__InlineList__"
+>
+  <ul
+    className="separate-items-with-comma"
+  >
+    <li
+      key="Map { \\"email\\": \\"johndoe@yahoo.com\\", \\"name\\": \\"John\\" }"
```

Comment:
I'd use ``email`` as a key

Reply:
actually, email or name because both are optional and it will render with one or the other